_data/accessibility-tests/pagination.yml

```diff
@@ -0,0 +1,67 @@
+title: Pagination
+component_status: pass
+test_items:
+  # General tests
+  - summary: Color is not the only method used to convey pagination link states.
+    summary_additional: When navigating through the pagination links, you can read the numbers to understand what page you are on. There may also be underlining missing to indicate you are on the current page.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.1
+  - summary: Numbers and background in in pagination links meet color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the pagination, contrast between the number color and background color is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.3
+  - summary: Pagination clearly indicates location on the website.
+    summary_additional: When viewing the paginated content, the numbers have distinct styling that give clear visual indication of where you are in the site and how to navigate back and forth.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.8
```

Comment:
**Question:** We recently decided to update this criterion as `conditional` for the breadcrumb component. Since users are required to manually set current pages and include missing pages via ellipses per the guidance, does it make sense to mark this as conditional as well?

Or do these tests assume correct implementation of the component, including this recommendation from the guidance? 

@amycole501 @alex-hull

Reply:
I would mark as conditional! We don't know the full implementation of this for other's sites, so we want to ensure this is being tested on the user's end.

Reply:
Resolved in [87d65fd](https://github.com/uswds/uswds-site/pull/2865/commits/87d65fdbbc1f87401f92a3f5131851f640456dc1)